pkg/ebpf/tracee.go

```diff
@@ -1251,6 +1322,20 @@ func (t *Tracee) Close() {
 			fmt.Fprintf(os.Stderr, "failed to clean containers module when closing tracee: %s", err)
 		}
 	}
+
+	if t.cgroupv1 != nil {
+		err := t.cgroupv1.Destroy()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to clean cgroupv1: %s", err)
```

Comment:
There's a logger now, let's use it for new code. If I missed other prints in the code it goes for that as well.

Reply:
Yep, I had doubts if I should just start using it for all the fprintfs around when refactoring, I believe we should as well.

Reply:
I would like this change to be made for all fprintfs and outside the scope of this PR.